app/assets/javascripts/discourse/app/components/bootstrap_mode_notice.js

```diff
@@ -0,0 +1,31 @@
+import GlimmerComponent from "@glimmer/component";
+import { htmlSafe } from "@ember/template";
+import I18n from "I18n";
+import { inject as service } from "@ember/service";
+
+export default class BootstrapModeNotice extends GlimmerComponent {
+  @service router;
+  @service siteSettings;
+  @service site;
+
+  get showBootstrapModeNotice() {
```

Comment:
This is no longer needed now that the check is in the application controller.

Reply:
Oh yeah, that was dumb of me to leave it hanging here. Removed 🔥